doc-src/templates/api-versions/model_documentor.rb

```diff
@@ -93,11 +93,19 @@ def initialize(operation_name, operation, api, klass)
     @lines += shapes(api, operation['input']).map {|line| "  " + line }
 
     ## @example tag
-
     @lines << "@example Calling the #{method_name(operation_name)} operation"
     @lines << generate_example(api, klass, method_name(operation_name),
                 operation['input']).split("\n").map {|line| "  " + line }
-    @lines << ""
+    if examples
+      examples.each do |example|
+        @lines << "@example Example: #{example['title']}"
```

Comment:
Example tags should not be prefixed with "Example:" text, let's just list the title.